osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/main/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/mbus/CoupleMBusDeviceCommandExecutor.java

```diff
@@ -87,7 +92,7 @@ public MbusChannelElementsResponseDto execute(final DlmsConnectionManager conn,
          * M-Bus slave device.
          */
         final ChannelElementValuesDto updatedChannelElementValues = this.deviceChannelsHelper
-                .writeUpdatedMbus(conn, requestDto, emptyChannelMatch.getChannel());
+                .writeUpdatedMbus(conn, requestDto, emptyChannelMatch.getChannel(), deviceContext);
```

Comment:
the idea was to prevent many methods to have passed the DlmsDevice, avoiding an increase in parameters to many methods. If the agreement was made to use inheritance then I will follow that path.